compiler+runtime/include/cpp/jank/runtime/obj/persistent_hash_map.hpp

```diff
@@ -10,6 +10,7 @@ namespace jank::runtime::obj
   using persistent_array_map_ptr = native_box<struct persistent_array_map>;
   using transient_hash_map_ptr = native_box<struct transient_hash_map>;
   using persistent_hash_map_ptr = native_box<struct persistent_hash_map>;
+  using meta_option = option<object_ptr>;
```

Comment:
I went with this approach since I assume we don't want to unwrap the option, only propagate it forward.

Reply:
Yeah, option makes sense. Let's remove the type alias, though. The others are there because they're forward declaring (the `struct` portion). Just use `option<object_ptr>` everywhere instead.